src/lib/rustContext.ts

```diff
@@ -56,7 +57,7 @@ export default class RustContext {
     // We need this await here, DO NOT REMOVE it even if your editor says it's
     // unnecessary. The constructor of the module is async and it will not
     // resolve if you don't await it.
-    const ctxInstance = await new this.rustInstance.Context(
+    const ctxInstance = new this.rustInstance.Context(
```

Comment:
wait dont remove this! see the comment! its a wasm binding issue we need the await, but it doesnt realize!

Reply:
100% your PR will be fucked up w out this so i have no worries of someone removing it

Reply:
Oh geez how did I miss this comment. Thank you, I'm putting it back.

Reply:
its okay your playwright would have been flakey as fuck an dyou would ahve realized at some point ahaha